musica/mechanism_configuration.py

```diff
@@ -93,6 +96,20 @@ def __init__(
         self.tracer_type = tracer_type if tracer_type is not None else self.tracer_type
         self.other_properties = other_properties if other_properties is not None else self.other_properties
 
+    @staticmethod
+    def serialize(cls):
+        return remove_empty_keys({
+            'name': cls.name,
+            'HLC_298K_mol_m3_Pa': cls.HLC_298K_mol_m3_Pa,
+            'HLC_exponential_factor_K': cls.HLC_exponential_factor_K,
+            'diffusion_coefficient_m2_s': cls.diffusion_coefficient_m2_s,
+            'N_star': cls.N_star,
+            'molecular_weight_kg_mol': cls.molecular_weight_kg_mol,
+            'density_kg_m3': cls.density_kg_m3,
+            'tracer_type': cls.tracer_type,
+            'other_properties': cls.other_properties,
+        })
+
```

Comment:
I was expecting these keys to match what we have in our [validation](https://github.com/NCAR/MechanismConfiguration/blob/main/include/mechanism_configuration/v1/validation.hpp) file for mechanism configuration. 

Lastly everything in `other_properties` would need to be serialized out as key value pairs. The code below may not be correct python, but it's at least pseduo code showing what I'm expecting. I would also expect this on the other classes

```suggestion
    @staticmethod
    def serialize(cls):
        d =  remove_empty_keys({
            'name': cls.name,
            'HLC(298K) [mol m-3 Pa-1]': cls.HLC_298K_mol_m3_Pa,
            'HLC(298K) [mol m-3 Pa-1]': cls.HLC_exponential_factor_K,
            'diffusion coefficient [m2 s-1]': cls.diffusion_coefficient_m2_s,
            'N star': cls.N_star,
            'molecular weight [kg mol-1]': cls.molecular_weight_kg_mol,
            'density [kg m-3]': cls.density_kg_m3,
            'tracer type': cls.tracer_type,
        })
        
        for key, val in cls.other_properties:
            d[key] = val
            
        return d
```